src/openloco/game_commands.h

```diff
@@ -9,27 +9,29 @@ using namespace openloco::interop;
 
 namespace openloco::game_commands
 {
-    enum GameCommandFlag
-    {
-        apply = 1 << 0,  // 0x01
-        flag_2 = 1 << 2, // 0x04
-        flag_3 = 1 << 3, // 0x08
-        flag_4 = 1 << 4, // 0x10
-        flag_5 = 1 << 5, // 0x20
-        flag_6 = 1 << 6, // 0x40
+    enum GameCommandFlag : uint32_t
+    {
+        apply = 1 << 0,     // 0x01
+        flag_2 = 1 << 2,    // 0x04
+        flag_3 = 1 << 3,    // 0x08
+        flag_4 = 1 << 4,    // 0x10
+        flag_5 = 1 << 5,    // 0x20
+        flag_6 = 1 << 6,    // 0x40
+        failure = 1U << 31, // 0x80000000
```

Comment:
Is this how we want to do this? It's not really a flag its a value you compare to. All of the other flags fit into one `uint8_t`.

Reply:
Ah, I didn’t realise. Let’s make it a constant, then. Suggestions for a name?